aspnetcore/blazor/security/webassembly/standalone-with-identity.md

```diff
@@ -237,6 +237,44 @@ In the `Backend` server API's `Program` file, a [Minimal API](xref:fundamentals/
 
 The roles endpoint requires authorization by calling <xref:Microsoft.AspNetCore.Builder.AuthorizationEndpointConventionBuilderExtensions.RequireAuthorization%2A>. If you decide not to use Minimal APIs in favor of controllers for secure server API endpoints, be sure to set the [`[Authorize]` attribute](xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute) on controllers or actions.
 
+## Cross-domain hosting (same-site configuration)
+
+The [sample apps](#sample-apps) are configured for hosting both apps at the same domain. If you host the `Backend` app at a different domain than the `BlazorWasmAuth` app, uncomment the code that configures the cookie (<xref:Microsoft.Extensions.DependencyInjection.IdentityServiceCollectionExtensions.ConfigureApplicationCookie%2A>) in the `Backend` app's `Program` file. The default values are:
+
+* Same-site mode: <xref:Microsoft.AspNetCore.Http.SameSiteMode.Lax?displayProperty=nameWithType>
+* Secure policy: <xref:Microsoft.AspNetCore.Http.CookieSecurePolicy.SameAsRequest?displayProperty=nameWithType>
+
+Change the values to:
+
+* Same-site mode: <xref:Microsoft.AspNetCore.Http.SameSiteMode.None?displayProperty=nameWithType>
+* Secure policy: <xref:Microsoft.AspNetCore.Http.CookieSecurePolicy.Always?displayProperty=nameWithType>
+
+```diff
+- options.Cookie.SameSite = SameSiteMode.Lax;
+- options.Cookie.SecurePolicy = CookieSecurePolicy.SameAsRequest;
++ options.Cookie.SameSite = SameSiteMode.None;
++ options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+```
+
+For more information on same-site cookie settings, see the following resources:
+
+* [Set-Cookie: `SameSite=<samesite-value>` (MDN documentation)](https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value)
+* [Cookies: HTTP State Management Mechanism (RFC Draft 6265, Section 4.1)](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-03#section-4.1)
+
+## Antiforgery support
+
+Only the logout endpoint (`/logout`) in the `Backend` app requires attention to mitigate the threat of [Cross-Site Request Forgery (CSRF)](xref:security/anti-request-forgery).
+
+The logout endpoint checks for an empty body to prevent CSRF attacks. By requiring a body, the request must be made from JavaScript, which is the only way to access the authentication cookie. The cookie can't be accessed by a form-based POST. This prevents a malicious site from logging the user out.
```

Comment:
```suggestion
The logout endpoint checks for an empty body to prevent CSRF attacks. By requiring a body, the request must be made from JavaScript, which is the only way to access the authentication cookie. The logout endpoint can't be accessed by a form-based POST. This prevents a malicious site from logging the user out.
```

While a cross-domain form-based POST doesn't exactly "access" the cookie, it will definitely include it as part of the request with `SameSiteMode.None`. The key is that the `/logout` endpoint will not be triggered by a form-based POST even though it will include the cookie.